libs/features/georoutingv2/georoutingv2.css

```diff
@@ -7,6 +7,7 @@
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
+  font-family: 'Adobe Clean', adobe-clean, 'Trebuchet MS', sans-serif;
```

Comment:
Hi @overmyheadandbody we cannot use  variable `--body-font-family` because the value for this variable will be adobe-clean-han-simplified-c or adobe-clean-han-traditional if the page has that language. We set value of the variable here https://github.com/adobecom/milo/blob/main/libs/styles/variables.css#L122 for this and we want to load adobe clean for all georouting modals irrespective of font on page.

Reply:
Ok, thanks for the context, makes sense!